modules/ROOT/pages/metadata-input.adoc

```diff
@@ -12,6 +12,8 @@ while Sources, Configurations and Connections will always have static metadata f
 
 == Declaring an InputTypeResolver
 
+The `InputTypeResolver` assigns a `MetadataType` to a class to resolve the input of the component.
+
 An `InputTypeResolver` implementation handles the request for resolving a `MetadataType` based
```

Comment:
Maybe add here a hyperlink to https://docs.mulesoft.com/mule-sdk/1.1/metadata#what-is-the-metadata-of-a-component in the MetadataType word.